packages/navbar/__tests__/navbar.spec.tsx

```diff
@@ -69,6 +69,20 @@ describe('<UiNavbar />', () => {
     expect(screen.getByText('Option 3')).toBeVisible();
   });
 
+  it('Should render navbar when roundedCorners is used', () => {
+    render(
+      <UiNavbar category="secondary" roundedCorners>
+        <UiNavbarItem>Option 1</UiNavbarItem>
+        <UiNavbarItem active>Option 2</UiNavbarItem>
+        <UiNavbarItem>Option 3</UiNavbarItem>
+      </UiNavbar>
+    );
+
+    expect(screen.getByText('Option 1')).toBeVisible();
+    expect(screen.getByText('Option 2')).toBeVisible();
+    expect(screen.getByText('Option 3')).toBeVisible();
```

Comment:
Just a question, are these tests capable to actually assert that it should render rounded corners or not, or is the intention to just make sure it will render fine when the prop is passed ?

Reply:
That's a good question, so short answer nope, although we could check using the helper `toHaveStyleRule` e.g. https://github.com/inavac182/uireact/blob/0914e4fdd5795ca7541c2809b33971ee0fcbfda5/packages/foundation/__tests__/spacing/spacing.spec.tsx#L27 Although I haven't been very diligent on testing that 😂  we will need to allocate some time to add unit tests for the styling of the components.

Reply:
The other idea would be to use visual tests